javav2/usecases/creating_photo_analyzer_app/Readme.md

```diff
@@ -440,36 +445,58 @@ The following Java code represents the **PhotoController** class that handles HT
        InputStream excelData = excel.exportExcel(myList);
 
        try {
-           // Email the report
+           //email the report
            sendMessage.sendReport(excelData, email);
 
        } catch (Exception e) {
 
            e.printStackTrace();
        }
-        return "The photos have been analyzed and the report is sent.";
+        return "The photos have been analyzed and the report is sent";
     }
 
-    // Upload an image to send to an S3 bucket
+    //Upload a video to analyze
```

Comment:
I can't tell if that's a capital or lowercase 'U' but it should be capital and there should be a period at the end of the sentence.